src/Traits/MpRequest.php

```diff
@@ -22,6 +23,11 @@ public function payment(): PaymentResource
         return new PaymentResource($this);
     }
 
+    public function authorized_payment(): AuthorizedPaymentResource
```

Comment:
Property and method names must follow the `camelCase` pattern, rename to `Invoice` and change the return type to `InvoiceResource`

Reply:
Actually, the resources are using `PascalCase`, in `camelCase` they should call `paymentResource`, `invoiceResource`.  The methods can be using `camelCase` or `snake_case`, as we don't have any one with two names to compare. But there is no problem ,I'll follow the pattern!

Reply:
Class names must be `PascalCase`, property and method names must follow `camelCase`.

Note: We will only use `camelCase` if necessary, this is not the case in this feature.